Simbody/include/simbody/internal/SimbodyMatterSubsystem.h

```diff
@@ -1344,6 +1344,20 @@ void multiplyByMInv(const State&    state,
                     const Vector&   v,
                     Vector&         MinvV) const;
 
+/**
+Multiply by the square root of the inverse mass matrix.
+Ref: Jain, Park, Vaidehi, J. Chem. Theory Comput., 8(8): 2581-2587. 2012.
+This operator calculates in O(n) time the product sqrt(M^-1)*v where M is the
+system mass matrix inverse and v is a supplied vector with one entry per u-space
+mobility. If v is a set of modal velocities drawn from a independent multivariate
+Gaussian distribution,  the result is a generalized velocity (udot=sqrt(M^-1)*v) vector
```

Comment:
This comment doesn't seem quite right -- it is written to return a udot (generalized acceleration) but did you mean it to return a u (generalized velocity)?

Also this line appears to be over 80 characters; should limit to 80 per coding guidelines.